gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.5-all.zip
```

Comment:
should be 5.5.1

Reply:
Looking at the fixes in .1 it doesn't seem like it'd affect us, but I've updated the wrapper just in case :-)